src/SprykerEco/Zed/Unzer/Business/Checkout/UnzerCheckoutHookExecutorInterface.php

```diff
@@ -16,7 +16,7 @@ interface UnzerCheckoutHookExecutorInterface
      * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
      * @param \Generated\Shared\Transfer\CheckoutResponseTransfer $checkoutResponseTransfer
      *
-     * @return void
+     * @return \Generated\Shared\Transfer\CheckoutResponseTransfer
      */
-    public function execute(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponseTransfer): void;
+    public function execute(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponseTransfer): CheckoutResponseTransfer;
```

Comment:
Please discuss this with the technical lead that this change is not a business interruption. And check please similar changes.

Reply:
There will be TcL review, imho this should be fine.

Reply:
In consultation with Sergey the comment can be resolved by me.